src/server/utils/onModuleLoad.js

```diff
@@ -71,7 +72,7 @@ export function validateCspIsPresent(csp) {
   }
 }
 
-/* eslint complexity:  ['error', 12] */
+/* eslint complexity:  ['error', 13] */
```

Comment:
consider:

```
if (moduleName === serverStateConfig.rootModuleName) {
    handleRootModuleConfig(module);
    return;
}
```

for a bit less complexity? (in both PR's if this function is the same between them?